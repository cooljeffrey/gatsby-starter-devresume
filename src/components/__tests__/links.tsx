import React from "react";
import renderer from "react-test-renderer";
import Links from "../links";
describe("Links", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(
        <Links
          links={{
            github: "test",
            linkedin: "testlinkedin",
            twitter: "twittertest",
          }}
        />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
