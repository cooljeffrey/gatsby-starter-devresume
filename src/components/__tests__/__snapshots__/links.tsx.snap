// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Links renders correctly 1`] = `
<div
  className="resume-footer text-center"
>
  <ul
    className="resume-social-list list-inline mx-auto mb-0 d-inline-block text-muted"
  >
    <li
      className="list-inline-item mb-lg-0 mr-3"
    >
      <a
        className="resume-link"
        href="https://github.com/test"
      >
        <i
          className="fab fa-github-square fa-2x mr-2"
          data-fa-transform="down-4"
        />
        <span
          className="d-none d-lg-inline-block text-muted"
        >
          github.com/
          test
        </span>
      </a>
    </li>
    <li
      className="list-inline-item mb-lg-0 mr-3"
    >
      <a
        className="resume-link"
        href="https://linkedin.com/in/testlinkedin"
      >
        <i
          className="fab fa-linkedin fa-2x mr-2"
          data-fa-transform="down-4"
        />
        <span
          className="d-none d-lg-inline-block text-muted"
        >
          linkedin.com/in/
          testlinkedin
        </span>
      </a>
    </li>
    <li
      className="list-inline-item mb-lg-0 mr-lg-3"
    >
      <a
        className="resume-link"
        href="https://twitter.com/twittertest"
      >
        <i
          className="fab fa-twitter-square fa-2x mr-2"
          data-fa-transform="down-4"
        />
        <span
          className="d-none d-lg-inline-block text-muted"
        >
          @
          twittertest
        </span>
      </a>
    </li>
  </ul>
</div>
`;
